### PR TITLE
python3Packages.twitterapi: init 2.6.6

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python wrapper for Twitter's REST and Streaming APIs

https://github.com/geduldig/TwitterAPI

This is a Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Depends on #112854 for Python 3.9.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113701` at 88265e3b run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>python39Packages.twitterapi</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/205b049917981eb1ba7a0934472a0531">python37Packages.twitterapi</a></li></li>
    <li><a href="https://gist.github.com/1a93ee607592f65b2057ac4d3e9ce431">python38Packages.twitterapi</a></li></li>
  </ul>
</details>

File: pkgs/development/python-modules/twitterapi/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    inherit version;
+    sha256 = "0k5spnghi92camkprpwhcz70wjbmhrbwmbp6prv77glnv0122vlb";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    requests_oauthlib
+  ];
+
+  # Tests are only available the moment
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/twitterapi/default.nix:23]: I don't understand.

Comment (fabaff) [reply pkgs/development/python-modules/twitterapi/default.nix:23]: Me neither.

**Review by JABERMQM — APPROVED** [review]

**Pull request merged**